QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/engine/install/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KubeStellar**](#1-install-and-run-kubestellar)
+- [2. Example deployment of Apache HTTP Server workload into two local kind clusters](#2-example-deployment-of-apache-http-server-workload-into-two-local-kind-clusters)
+  - [a. Stand up two kind clusters: florin and guilder](#a-stand-up-two-kind-clusters-florin-and-guilder)
+  - [b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#b-create-a-kubestellar-inventory-management-workspace-imw-and-workload-management-workspace-wmw)
+  - [c. Onboarding the clusters](#c-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#e-create-and-deploy-the-apache-server-workload-into-florin-and-guilder-clusters)
+- [3. Teardown the environment](#3-teardown-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+kubestellar-bootstrap.sh --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder .
+export PATH="$PATH:$(pwd)/kcp/bin:$(pwd)/kubestellar/bin"
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+Check that `KubeStellar` is running:
+
+First, check that controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
```

Comment:
That display now shows command line arguments that do not actually appear.

Reply:
which one? I don't understand this comment

Reply:
The `kubestellar start` command launches the controllers with the following command lines.

```shell
mailbox-controller -v=2 >& $log_folder/mailbox-controller-log.txt &
scheduler -v 2 >& $log_folder/edge-scheduler-log.txt &
placement-translator --allclusters-context  "system:admin" -v=2 >& $log_folder/placement-translator-log.txt &
```

The first two do not match what is shown here.  The third still has an unnecessary flag (`--allclusters-context  "system:admin"`).

Reply:
The command below is looking at the process names: 

```ps aux | grep -e mailbox-controller -e placement-translator -e scheduler```

I have used it several times and I am sure that it works well

Reply:
My point here is not about the command but about the exhibited output from the command.

Reply:
For example, consider the mailbox-controller.  The _output_ shown below shows command line arguments that used to be given but are not now.

Reply:
Braulio to fix this.

Reply:
update received - resolving